examples/bulk.php

```diff
@@ -71,7 +70,7 @@ function toJSON(object $document): string
     ]
 );
 
-$cursor = $collection->find([], ['batchSize' => 2]);
+$cursor = $collection->find([]);
```

Comment:
FWIW, `Collection::find()` can be called without arguments as `$filter` defaults to an empty array. This technically violates the CRUD spec but we snuck that past the goalie long ago. Up to you if you'd like to keep this as-is or change to `find()`.

Likewise in `with_transaction.php`.